We can't handle nested tlvs yet, so make these byte arrays.

diff --git b/wire/onion_wire.csv a/wire/onion_wire.csv
index 5c52fe9a1..2ac0c4cff 100644
--- b/wire/onion_wire.csv
+++ a/wire/onion_wire.csv
@@ -51,11 +29,11 @@ tlvdata,onionmsg_tlv,reply_path,path,onionmsg_path,...
 tlvtype,onionmsg_tlv,encrypted_data_tlv,4
 tlvdata,onionmsg_tlv,encrypted_data_tlv,encrypted_data_tlv,byte,...
 tlvtype,onionmsg_tlv,invoice_request,64
-tlvdata,onionmsg_tlv,invoice_request,invoice_request,tlv_invoice_request,
+tlvdata,onionmsg_tlv,invoice_request,invoice_request,byte,...
 tlvtype,onionmsg_tlv,invoice,66
-tlvdata,onionmsg_tlv,invoice,invoice,tlv_invoice,
+tlvdata,onionmsg_tlv,invoice,invoice,byte,...
 tlvtype,onionmsg_tlv,invoice_error,68
-tlvdata,onionmsg_tlv,invoice_error,invoice_error,tlv_invoice_error,
+tlvdata,onionmsg_tlv,invoice_error,invoice_error,byte,...
 subtype,blinded_path
 subtypedata,blinded_path,first_node_id,point,
 subtypedata,blinded_path,blinding,point,
